### PR TITLE
add run_as_service

### PR DESCRIPTION
**Pull request by cherokee94**

### CODE REVIEW

File: docs/modules/installation/pages/run_as_service.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+....
+sudo nano /etc/systemd/system/axion.service
+....
+The unit file should contain: 
++
+....
+[Unit]
+Description=axion
+Wants=postgresql.service
+After=network.target local-fs.target postgresql.service
```
Comment (arufanov) [inline docs/modules/installation/pages/run_as_service.adoc:18]: @cherokee94 
* this after assume that axion use postgres as database it is true for production. Should we say clear: this unit file assume that axion use pg?

Comment (cherokee94) [reply docs/modules/installation/pages/run_as_service.adoc:18]: Added a big NOTE after the unit file example

File: docs/modules/installation/pages/run_as_service.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+[Service]
+Type=simple
+User=axion
+WorkingDirectory=/home/axion/
+SyslogIdentifier=axion
+TimeoutSec=5min
+Restart=on-failure
+RestartSec=5s
+SuccessExitStatus=143
+ExecStart=java -jar /home/axion/axion-0.1.0.jar --application.keys.google_analytics=G-123456
```
[the remark this answers is not in the document]
Comment (cherokee94) [reply docs/modules/installation/pages/run_as_service.adoc:29]: Same NOTE from above

**Review by arufanov — APPROVED** [review]
@cherokee94 
thanks!

**Pull request merged**